redash/aws/ec2.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_instance" "redash-instance" {
+  ami = "ami-0c654c3ab463d22f6"
+  instance_type = "t2.small"
```

Comment:
will the user be able somehow to chose instance types?

Reply:
Not sure that it will be available for the MVP, but it would be probably in the next phases.